android/app/build/intermediates/manifest_merge_blame_file/debug/manifest-merger-blame-debug-report.txt

```diff
@@ -9,86 +9,86 @@
 9        android:targetSdkVersion="31" />
 10
 11    <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW" />
-11-->/home/ekoehler/Documents/leonart/mobile/android/app/src/debug/AndroidManifest.xml:5:5-77
-11-->/home/ekoehler/Documents/leonart/mobile/android/app/src/debug/AndroidManifest.xml:5:22-75
+11-->C:\Users\cross\Desktop\Epitech\EIP\Mobile\android\app\src\debug\AndroidManifest.xml:5:5-77
+11-->C:\Users\cross\Desktop\Epitech\EIP\Mobile\android\app\src\debug\AndroidManifest.xml:5:22-75
 12    <uses-permission android:name="android.permission.INTERNET" />
-12-->/home/ekoehler/Documents/leonart/mobile/android/app/src/main/AndroidManifest.xml:4:5-67
-12-->/home/ekoehler/Documents/leonart/mobile/android/app/src/main/AndroidManifest.xml:4:22-64
+12-->C:\Users\cross\Desktop\Epitech\EIP\Mobile\android\app\src\main\AndroidManifest.xml:4:5-67
+12-->C:\Users\cross\Desktop\Epitech\EIP\Mobile\android\app\src\main\AndroidManifest.xml:4:22-64
 13    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
-13-->[com.facebook.flipper:flipper:0.125.0] /home/ekoehler/.gradle/caches/transforms-3/03f04a3f11198ce3b7c6cedb9b41266b/transformed/jetified-flipper-0.125.0/AndroidManifest.xml:16:5-76
-13-->[com.facebook.flipper:flipper:0.125.0] /home/ekoehler/.gradle/caches/transforms-3/03f04a3f11198ce3b7c6cedb9b41266b/transformed/jetified-flipper-0.125.0/AndroidManifest.xml:16:22-73
+13-->[com.facebook.flipper:flipper:0.125.0] C:\Users\cross\.gradle\caches\transforms-3\03f04a3f11198ce3b7c6cedb9b41266b\transformed\jetified-flipper-0.125.0\AndroidManifest.xml:16:5-76
+13-->[com.facebook.flipper:flipper:0.125.0] C:\Users\cross\.gradle\caches\transforms-3\03f04a3f11198ce3b7c6cedb9b41266b\transformed\jetified-flipper-0.125.0\AndroidManifest.xml:16:22-73
 14
 15    <application
-15-->/home/ekoehler/Documents/leonart/mobile/android/app/src/main/AndroidManifest.xml:6:5-25:19
+15-->C:\Users\cross\Desktop\Epitech\EIP\Mobile\android\app\src\main\AndroidManifest.xml:6:5-25:19
 16        android:name="com.mobile.MainApplication"
-16-->/home/ekoehler/Documents/leonart/mobile/android/app/src/main/AndroidManifest.xml:7:7-38
+16-->C:\Users\cross\Desktop\Epitech\EIP\Mobile\android\app\src\main\AndroidManifest.xml:7:7-38
 17        android:allowBackup="false"
-17-->/home/ekoehler/Documents/leonart/mobile/android/app/src/main/AndroidManifest.xml:11:7-34
+17-->C:\Users\cross\Desktop\Epitech\EIP\Mobile\android\app\src\main\AndroidManifest.xml:11:7-34
 18        android:appComponentFactory="androidx.core.app.CoreComponentFactory"
-18-->[androidx.core:core:1.7.0] /home/ekoehler/.gradle/caches/transforms-3/63a3df243d2d102452600006144a86df/transformed/core-1.7.0/AndroidManifest.xml:24:18-86
+18-->[androidx.core:core:1.7.0] C:\Users\cross\.gradle\caches\transforms-3\63a3df243d2d102452600006144a86df\transformed\core-1.7.0\AndroidManifest.xml:24:18-86
```

Comment:
This kind of things is Windows specific. Please do not push them next time, we may face issues as Marine and I are on Unix-based systems.